docs/user/aws/install_upi.md

```diff
@@ -19,6 +20,29 @@ $ openshift-install-linux-amd64 create ignition-configs
 ? Pull Secret [? for help]
 ```
 
+Edit the resulting `openshift-install.yaml` to set `replicas` to 0 for the `compute` pool, which will remove the compute MachineSets.
```

Comment:
nit provide instructions of using their favorite editor to edit the specific file, and also example of how it would look.

Reply:
> nit provide instructions of using their favorite editor to edit the specific file

What's their favorite editor, `sed`? ;)

> and also example of how it would look.

If I make this a `sed` command, can I skip the example?  I want to minimize the examples we have to update whenever we bump something in the install-config.

Reply:
For in repo docs I think sed is fine provided it's easy enough to follow and we feel it won't create a headache to maintain. For openshift-docs we need to have an example. So decide if it's easier to have them look approximately the same or not.

Reply:
Since this is the only line we want them to change in the `install-config.yaml` file in this flow, I vote for `sed`.

Reply:
Ok, if you're fine with sed i'm fine with it.

Reply:
I've pushed a `sed` approach with 5e34eb130 -> cc7d56dea (also rebases onto master).  It's a bit fiddly, for reasons discussed in the cc7d56dea commit message.  Is it worth it?  I'm not sure what else we'd do for POSIX-compliant YAML edits, but we could always give an example in... Go?  Python?... that makes a YAML-aware edit if we don't mind leaving folks without whatever tool we pick to translate themselves instead of copy/pasting.